quinn-proto/src/tests/util.rs

```diff
@@ -515,6 +520,16 @@ fn packet_size(transmit: &Transmit) -> usize {
     transmit.contents.len()
 }
 
+fn ecn(x: Option<EcnCodepoint>, congestion_experienced: bool) -> Option<EcnCodepoint> {
+    x.map(|codepoint| {
+        if congestion_experienced {
```

Comment:
Style: would use `match` instead of `if` here.

Maybe the function name could be a bit more descriptive? `ecn_or()` or `ecn_and_then()` or something else...

Reply:
The method is setting the congestion experienced bit if ECN is supported and `congestion_experienced == true`. I agree that `ecn` is too generic. I'll use `set_congestion_experienced`, which is more descriptive.